includes/filerepo/file/LocalFileDeleteBatch.php

```diff
@@ -284,7 +284,7 @@ protected function doDBInserts() {
 				}
 			}
 
-			$dbw->insert( 'filearchive', $rowsInsert, __METHOD__ );
+			$dbw->insert( 'filearchive', $rowsInsert, __METHOD__, [ 'IGNORE' ] );
```

Comment:
OK - in upstream: https://github.com/Wikia/mediawiki/blob/2e236f056f193aa6702c42ffc4b2e678c2745eeb/includes/filerepo/file/LocalFileDeleteBatch.php#L293-L299